src/main/webapp/scripts/itinerary-overview.js

```diff
@@ -227,6 +225,50 @@ function displayRoute(start, waypoints, end) {
     );
 }
 
+/**
+ * Method that changes the travel mode that the user will be using on their trip.
+ * @param travelMode{String} one of the accepted modes of travel by displayRoute() (DRIVING,WALKING,BICYCLING,TRANSIT)
+ */
+function changeTravelMode(travelMode){
+    currentMode = travelMode;
+    displayRoute(start, waypoints, end);
+}
+
+function changeTravelModeDriving(){
+    changeTravelMode("DRIVING");
+}
+
+function changeTravelModeTransit(){
+    changeTravelMode("TRANSIT");
+}
+
+function changeTravelModeWalking(){
+    changeTravelMode("WALKING");
+}
+
+function changeTravelModeCycling(){
+    changeTravelMode("BICYCLING");
+}
+/**
+ * Gets URL for photo of place or assigns it a default one if there are no photos available
+ * @param {Array} photos array of photos from PlaceResult object
+ */
+async function imageURLFromPhotos(photos) {
+    const photoRef =
+        photos && Array.isArray(photos)
+            ? photos[0].photo_reference
+            : undefined;
+
+    if(photoRef) {
+        const photoResponse = await fetch(
+            `https://cors-anywhere.herokuapp.com/https://maps.googleapis.com/maps/api/place/photo?maxheight=300&photoreference=${photoRef}&key=${GOOGLE_API_KEY}`
+        );
+        const blob = await photoResponse.blob();
+        return URL.createObjectURL(blob);
+    } else {
+        return DEFAULT_PLACE_IMAGE;
+    }
+}
```

Comment:
This is currently one of the common util functions I'm refactoring so I may need to change this slightly in a future PR.